### PR TITLE
docs: Fix its/it’s, where appropriate

### PR DESCRIPTION
**Pull request by Zearin**

### CODE REVIEW

**Comment by Zearin** [comment]
@ByteHamster How bout it? ;)

File: source/dav/authentication.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ SabreDAV comes with the following backends:
 | Class                                  | Type   | Description |
 |--------------------------------- ----- | ------ | ----------- |
 | `Sabre\DAV\Auth\Backend\Apache`        | N/A    | Lets the webserver handle authentication |
-| `Sabre\DAV\Auth\Backend\BasicCallBack` | Basic  | Extremely easy way to create authentication from a custom source |
-| `Sabre\DAV\Auth\Backend\File`          | Digest | Use a `htdigest` file for it's backend |
+| `Sabre\DAV\Auth\Backend\BasicCallback` | Basic  | Extremely easy way to create authentication from a custom source |
```
Comment (ByteHamster) [inline source/dav/authentication.md]: Was this change intended? The class in sabre/dav has a capital letter: https://github.com/sabre-io/dav/blob/62cfe9d497652bf3a687f59776a792c8d1fceb86/lib/DAV/Auth/Backend/BasicCallBack.php

Comment (Zearin) [reply source/dav/authentication.md]: It was not intended.  I’ll fix it in a sec.

**Comment by ByteHamster** [comment]
Thanks!
